.github/dependabot.yml

```diff
@@ -1102,3 +1092,13 @@ updates:
     schedule:
       interval: "weekly"
       day: "wednesday"
+  - package-ecosystem: "gomod"
+    directory: "/receiver/zipkinreceiver"
```

Comment:
Are these being added because they were missing and shouldve been here the whole time?

Reply:
Correct... these are added because two items are removed from the top